File: src/app/models/uploadedFile.ts
export class UploadedFile{
	id: string;
	name: string
	size: number
	type: string
	path?: string;

	constructor(details: UploadedFile) {
		this.id = details.id
		this.name = details.name
		this.size = details.size
		this.type = details.type
		this.path = details.path		
	}
}

File: backend/Models/UploadedFile.cs
public class UploadedFile{
	public Guid Id { get; set; }
	public required string Name { get; set; }
	public required long Size { get; set; }
	public required string Type { get; set; }
	public string? Path { get; set; }

	// public Guid ConferenceId;

	public Guid ConferenceId { get; set; }
	public Conference Conference { get; set; } = null;

}